### ISSUE TITLE
Using Permalinks

### ISSUE THREAD

**Issue by viktorix** [ClassicPress/Documentation-Issue-Tracker#23]
https://wordpress.org/support/article/using-permalinks/

Moved from [Notion list](https://silly-plaster-ac8.notion.site/53802faf6be041619c9b5aa1ff513de6?v=d6671f35b580460b82cb74fd7e260cb1).

**Comment by MarkDavenport** [comment]
Hi,
I can copy content and change to march ClassicPress.
What about links e.g.  Plugin Directory which links to  https://wordpress.org/plugins/search/taxonomy/

**Comment by viktorix** [comment]
@MarkDavenport
1. We want to avoid duplicate SEO issues with WordPress docs. So we need to rewrite articles in our own words. So it's more than just changing WordPress to ClassicPress. That's why it's a bit slower.
2. That's a good question. I think the best option is to list any internal links that do not have a CP version here as a comment, so we can see if they need to be rewritten. In the post, I wouldn't add those links back linked to WordPress. Maybe marking the anchor text with square brackets like `[text]` would help us keep track of where we need to add links back. So the text would look like this:

> This sentence shows how [square brackets] can be used for a future URLs.

I will be writing contributor instructions. What I would like to do is give users an author role account in the docs site. So they can write the article (upload the photos) and save it as a draft, so we can review and publish it. 

I'm open to suggestions.

**Comment by KTS915** [comment]
Agreed on author role. Let's use CP the way it's designed to be used.

**Comment by MarkDavenport** [comment]
Author role would be good, and would save the "checker" having to do basic formatting and as you say uploading photos etc.
Guide lines on both image size and file size, and type ( on some of my clients site they have uploaded 4Mb 99% Jpegs or png screen shots etc.)

**Comment by viktorix** [comment]
Our docs site uses integration with the Discourse forum to enable SSO. I'm trying to figure out an issue with it, so I can create additional users. I will let you know soon once that's fixed. I will include guidelines in the instructions.

**Comment by viktorix** [comment]
@MarkDavenport I added you to a group in the forum cpnet-docs-authors. You should have received a notification. Go to the login page https://docs.classicpress.net/wp-login.php and try to log in with Discourse. You should be able to log in. If you run into any issues, please do let me know.

If you plan to work on Using Permalinks article, please create it as a User Guide. You'll see it as custom post type.

If there are any hyperlinks to WP articles. You can keep them, but do mention them here or create an issue for each one.

Also, follow proper headings H2, H3 for stuff within H2's section, and so on. This will be converted automatically to a table of contents (ToC). So proper usage allows for nesting/sub-items in the ToC.
